### PR TITLE
Cu 1qfvcee turn statistics into single type

### PR DESCRIPTION
**Pull request by bosung90**

### CODE REVIEW

**Comment by bosung90** [comment]
Task linked: [CU-1qfvcee Turn Statistics into Single Type](https://app.clickup.com/t/1qfvcee)

File: src/App.tsx [hunk 1]
```diff
@@ -36,6 +32,7 @@ export default function App() {
                     <Route exact path="/events" component={() => <EventsPage />} />
                     <Route exact path="/events/:id" component={() => <EventDetailsPage />} />
                     <Route exact path="/about-us" component={() => <AboutUsPage />} />
+                    <Route exact path="/jobs" component={() => <JobsPage />} />
```
Comment (bennettoh) [inline src/App.tsx:35]: nit: I think you can just do `component={JobsPage}`

File: src/pages/AboutUsPage.tsx [hunk 2]
```diff
@@ -134,7 +134,7 @@ const Member = ({name, title, description, profilePic, socialList}: MemberType)
                 }}>
                 {socialList?.map?.((sns, index) => (
                     <a key={index} href={sns.link} target="_blank" rel="noreferrer">
-                        <img src={memberSocialIcons?.[sns.type] ?? memberSocialIcons.link} alt={sns.type} />
+                        <img src={socialIcons?.[sns.type] ?? socialIcons.other} alt={sns.type} />
```
Comment (bennettoh) [inline src/pages/AboutUsPage.tsx:137]: socialIcons will never be undefined right?

File: src/pages/JobsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            <NavigationBar />
+            <Section spacing={8} mt={5}>
+                <Stack>
+                    <Typography>Hello</Typography>
+                </Stack>
+            </Section>
+            <Footer />
+        </>
+    )
+}
```
Comment (bennettoh) [inline src/pages/JobsPage.tsx:20]: We should move job related code to another PR if we want to be clean 👀

File: src/store.ts [hunk 4]
```diff
@@ -1,42 +1,54 @@
 import React from 'react'
-import {getPhotos, getSponsors, getStats, getEvent, getEvents, getMembers} from 'api'
-import type {PhotoType, MemberType, SponsorsType, UseDocument, UseCollection, EventType, StatisticsType} from 'types'
+import {getPhotos, getSponsors, getStatistics, getEvent, getEvents, getOrganizers, getJobs, getJob} from 'api'
+import {getFooter} from 'api'
+import type {PhotoType, OrganizerType, SponsorsType, UseDocument, UseCollection, EventType, StatisticsType} from 'types'
+import type {JobType, FooterType} from 'types'
```
Comment (bennettoh) [inline src/store.ts:5]: combine imports?

**Review by bennettoh — APPROVED** [review]

File: src/store.ts (hunk 4 above)
Comment (bosung90) [reply src/store.ts:5]: I didn't want the import to go into multiple lines

File: src/pages/JobsPage.tsx (hunk 3 above)
Comment (bosung90) [reply src/pages/JobsPage.tsx:20]: I agree, but I'll work on it on next PR :)
